### DISCUSSION TITLE
2.6.4win11报告有毒

### DISCUSSION THREAD

**Discussion by ironstraight** [AlistGo/alist#1512] [Q&A]
2.6.4win11报告arm64的有毒，请问新版安全吗？？  报告内容Trojan:Win32/Wacatac.B!ml

**Comment by cxw620** [comment]
> 2.6.4win11报告arm64的有毒，请问新版安全吗？？ 报告内容Trojan:Win32/Wacatac.B!ml

不交保护费是这样的. 你确保是从Github的release下载的就行
